zulipterminal/ui_tools/views.py

```diff
@@ -177,12 +177,21 @@ def update_search_box_narrow(self, message_view: Any) -> None:
             )
         self.model.controller.update_screen()
 
-    def read_message(self) -> None:
+    def read_message(self, index: int=-1) -> None:
```

Comment:
This `index` seems unused?

Reply:
Correct me if I am wrong, but this seems to be a default parameter passed with the callback which is being caught with 'index' variable?

Reply:
Yes Sumanth it is. @neiljp `index` will have to remain unused.

Reply:
To clarify, this is an expected parameter for a callback method provided to `set_focus_changed_callback`?
(this seems not well documented?)

Reply:
yes.